dkg/dkg.go

```diff
@@ -109,8 +109,8 @@ func Run(ctx context.Context, conf Config) (err error) {
 	}
 
 	// This DKG only supports a few specific config versions.
-	if def.Version != "v1.6.0" && def.Version != "v1.7.0" {
-		return errors.New("only v1.6.0 and v1.7.0 cluster definition version supported")
+	if def.Version != "v1.6.0" && def.Version != "v1.7.0" && def.Version != "v1.8.0" {
```

Comment:
Wonder if we should only support v1.8.0?

Reply:
This breaks tests like `TestDKG/frost_v16` that aimed for older versions.